testproject/ProjectSettings/EditorBuildSettings.asset

```diff
@@ -8,6 +8,9 @@ EditorBuildSettings:
   - enabled: 1
     path: Assets/MainMenu.unity
     guid: 21c7fd68cda9dd9488fc0cc01f885e39
+  - enabled: 1
+    path: Assets/Scenes/ZooSam.unity
```

Comment:
"ZooSam"? LOL

Reply:
hehe you never used Zoos?
It's mostly a way to have our own sandboxes, without polluting one project with each other's experiments. This way I have my own little controlled environment, I know exactly what's in there and what settings I'm using.